examples/aws-on-prem/README.md

```diff
@@ -0,0 +1,59 @@
+# AWS Log Source integration
+
+This example creates an Elastic Stack (Elasticsearch and Kibana),
+and all necessary AWS resources to ingest AWS log sources.
+[AWS CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html) is
+the example log source.
+
+This example implements the [Official Elastic's Blog Post on AWS Log Source ingestion](https://www.elastic.co/blog/getting-aws-logs-from-s3-using-filebeat-and-the-elastic-stack) in Terraform and Kubernetes.
+
+## Schematic
+
+![cloudtrail-schematic](https://github.com/operatorequals/Blue-Baron/raw/main/assets/aws-integration.svg)
```

Comment:
Schematic Image doesn't show. Add `img![cloudtrail-schematic](https://github.com/operatorequals/Blue-Baron/raw/main/assets/aws-integration.svg)`

Reply:
It will show when it gets to `main` branch.